### DISCUSSION TITLE
Selecting the Civitai button no longer loads in the Civitai Browser+ extension?

### DISCUSSION THREAD

**Discussion by SmolJacketPocket** [BlafKing/sd-civitai-browser-plus#205] [Q&A]
I'm not sure when the change happened or if there's an overlap in other extensions, but in the video it shows that when you click the civitai icon in the Lora, checkpoint, etc thumbnail, it'll take to you the CivitAI Browser+ extension and load it there. Now, it loads a floating html page. Is there a setting that I need to enable to restore this behavior or was this a feature change?

I've tested with and without the Lobe theme. I'm using v1.7.0 Automatic1111, gradio 3.41.2 and the latest CivitAI Browser+ as of 10/2/24. 

Here's what I see:
![civitaibrowser](https://github.com/BlafKing/sd-civitai-browser-plus/assets/160309082/4788a7e5-54d1-47cb-bb55-a697c21faf0a)

**Comment by BlafKing** [comment]
This behavior has indeed changed from when the preview video was created, the button now acts as a pop up HTML with the info of the model instead of loading it in the browser.

I could add a setting that keeps the old behavior of loading the model in the browser, if you would prefer to have that back!

**Reply by SmolJacketPocket** [reply]
Please! That would be great! And thanks for responding so fast! 

EDIT: Or rather a selection/check box just in case others like the new behavior.

**Reply by BlafKing** [reply]
I'll add it to the to-do list 👍🏻 :)
